INSTALL.md

```diff
@@ -63,207 +63,277 @@ cfgov assets
-Start MYSQL with the following command:
-```
-mysql.server start
-```
+#### Front-end dependencies
 
-Then run the MYSQL creation script from project root directory:
-```
-./create-mysql-db.sh
-```
+The cfgov-refresh front end currently uses the following frameworks / tools:
 
-If you would like to have a custom database setup
-then you can pass in the necessary arguments:
+- [Gulp](http://gulpjs.com): task management for pulling in assets,
+  linting and concatenating code, etc.
+- [Bower](http://bower.io): Package manager for front-end dependencies.
+- [Less](http://lesscss.org): CSS pre-processor.
+- [Capital Framework](https://cfpb.github.io/capital-framework/getting-started):
+  User interface pattern-library produced by the CFPB.
 
-```
-./create-mysql-db.sh <dbname> <username> <password>
+> **NOTE:** If you’re new to Capital Framework, we encourage you to
+  [start here](https://cfpb.github.io/capital-framework/getting-started).
+
+1. Install [Node.js](http://nodejs.org) however you’d like.
+   We recommend using [nvm](https://github.com/creationix/nvm), though.
+
+   _NOTE: This project requires Node.js v5.5 or higher, and npm v3 or higher._
+2. Install [Gulp](http://gulpjs.com) and [Bower](http://bower.io):
+
+```bash
+npm install -g gulp bower
 ```
 
-> **NOTE:** Be sure to update your local settings in
-  `cfgov/cfgov/settings/local.py` to account for these changes.
 
+### 2. Run `setup.sh`
 
-If something goes wrong and you'd like to delete the database
-and start again, you can do so with:
+At this point, your machine should have everything it needs to automate the
+rest of the setup process.
 
+If you haven't cloned this repo yet, clone it to a local folder.
+Because related projects will need to be installed as siblings to this project,
+we recommend putting them all in their own folder, e.g., `~/Projects/cf.gov`.
+
+Once cloned, from the project root (`~/Projects/cf.gov/cfgov-refresh/`),
+run this command to complete the setup process:
+
+```bash
+source setup.sh
 ```
-mysql v1 -u root -p -e 'drop database v1;'
-```
 
-#### Virtual Environment
-In the project root directory,
-create a virtualenv that you’ll name `cfgov-refresh`:
+This will take several minutes, going through the steps in these scripts:
+
+1. `env-var-setup.sh`
+1. `frontend.sh`
+1. `backend.sh`
+
+Once complete, you should have a fully functioning local environment,
+ready for you to develop against!
+
+Want to know more about what the setup scripts are doing?
+[Read the detailed rundown.](#curious-about-what-the-setup-scripts-are-doing)
+
+Get any errors? [See our troubleshooting tips.](#troubleshooting)
+
+**Continue following the [usage instructions](README.md#usage) in the README.**
+
+
+## Optional steps
+
+### Load a database dump from the Build server
+
+If you're installing this fresh, the initial data you receive will not be
+as extensive as you'd probably like it to be.
+You can get a database dump from the Build server by running the Jenkins job
```

Comment:
This could use some more context for where to go to find the Jenkins job.. if that can be placed in public GH.